### ISSUE TITLE
Add confirmation on unsaved report

### ISSUE THREAD

**Issue by AlexKatyukov** [thewca/worldcubeassociation.org#4652]
**Problem:**
I was making a report and accidentally click to "Clone". New page was opened, "Back" button does not helped, I missed it all.

**Solution:**
Add JS Alarm reminder that there is unsaved changes like on Manage schedule page.

**Comment by Jambrose777** [comment]
Manually using the report editor to type the report can typically lead to hardships like what you experienced. (also accidentally clicking back button or having 2 delegates click save will overwrite the other's work).
I'd recommend typing up the report in a google doc then copy pasting it in the editor.

**Comment by AlexKatyukov** [comment]
Yes, I think that it's good idea to use Google Docs. But JS alarm here is not bad and hadr idea here :)

**Comment by dunkOnIT** [comment]
Duplicate of #7400 (which was created later but has a better spec for the work)
